ajax/libs/tiny-date-picker/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "tiny-date-picker",
+  "description": "A small, dependency-free date picker",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/chrisdavies/tiny-date-picker.git"
+  },
+  "keywords": [
+    "date",
+    "picker",
+    "date",

```

Comment:
Duplicate keywork with https://github.com/ArmenAvetisyan/cdnjs/blob/89d7bae581e33ea71b255bdb182d524fdc3a0d28/ajax/libs/tiny-date-picker/package.json#L9. Let's remove it.